### DISCUSSION TITLE
GitHub On Demand Training 102

### DISCUSSION THREAD

**Discussion by denicesp** [community/community#21870] [New to GitHub]
<p>I keep failing the Travis CI Build for GitHub On Demand Training 102. I’ve attached a screenshot of the file I committed in my Pull Request. Would someone please advise? Thank you. <img src="https://user-images.githubusercontent.com/33042373/181067362-acfb9d62-153a-4b27-bad5-94f7ce4a2008.png" alt="GitHub_102.png" data-base62-sha1="rlL1g13v7ehC5aIJaV5PMLBWEwa" width="689" height="143"></p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/denicesp">@denicesp</a>,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Cheers!</p>

**Comment by hectorsector** [comment, marked as answer]
<p>:wave: <a class="mention" href="/denicesp">@denicesp</a> sorry about that! The tests on the course, test wether all previous web pages still work. It turns out a <a href="https://travis-ci.org/githubschool/on-demand-github-pages/builds/375998739#L2131" rel="nofollow noopener">previous page</a> wasn’t working. I’ve pushed a fix to your branch and the tests should now pass. </p>
